modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -20005,4 +20005,9 @@ zip=Zip
 zip-postal=Zip/Postal
 zoom-in=Zoom In
 zoom-out=Zoom Out
-zoom-to-fit=Zoom to Fit
\ No newline at end of file
+zoom-to-fit=Zoom to Fit
+commerce-cookies-rejected=Cookies Rejected:
+commerce-cookies-rejected-description=Without cookies acceptance it’s not possibile to use compare function
+commerce-cookies-accepted=Cookies Accepted: 
+commerce-cookies-accepted-description=It’s now possibile to use the compare function
```

Comment:
The same thing here.

Reply:
Fixed

Reply:
`buildLang` keeps correcting the text to 

> It is